apis/userapi/userapi.yaml

```diff
@@ -1799,6 +1891,28 @@ definitions:
             type: string
             description: userId
 
+  MigrateUserReq:
+    description: request body for migrating user
+    type: object
+    properties:
+      request:
+        type: object
+        properties:
+          userId:
+            type: string
+            description: userId of migrating user
```

Comment:
Identifier of the user that needs to be migrated to a new tenant.

Reply:
done